packages/compartment-mapper/README.md

```diff
@@ -327,7 +331,7 @@ one workflow to the beginning of another, either as bytes or a location.
      map compartments ->  * *  * *   .'.| | |' : :
          read archive ->  | |  | |  '   * * *  : :
        unpack archive ->  | |  | |  :   * * *  : :
-assemble compartments ->  * *  * *  :   + + *  : : <- endowments
+assemble compartments ->  * *  * *  :   + + *  : : <- powers
```

Comment:
I'm digging the change from `endowments` to `globals`, but I have to assume that the word "powers" here means something different? Is it a superset of `globals` plus something else? I see "powers" used in a few other places, but (at least in this diff) I'm not seeing a definition.

Should the prose above this point mention what "powers" are if they aren't just the same as "globals"?

Reply:
I’m using “powers” to mean `{ modules, globals, globalLexicals }` to avoid confusing the argument named `endowments` on the `Compartment` constructor. But, that itself might be confusing.